src/components/SaleCard/SaleCard.stories.tsx

```diff
@@ -0,0 +1,159 @@
+import { Meta } from '@storybook/react';
+import SaleCard, { SaleCardProps } from './SaleCard';
+import { getScssVar } from '../../utils/scssUtils';
+
+const meta = {
+  title: 'Components/SaleCard',
```

Comment:
can we change this to Patterns/SaleCard
```suggestion
  title: 'Patterns/SaleCard',
```